netbox/utilities/choices.py

```diff
@@ -230,3 +230,17 @@ class ImportFormatChoices(ChoiceSet):
         (JSON, 'JSON'),
         (YAML, 'YAML'),
     ]
+
+
+class CSVDelimiterChoices(ChoiceSet):
+    AUTO = 'auto'
+    COMMA = ','
+    SEMICOLON = ';'
+    TAB = '\t'
+
+    CHOICES = [
+        (AUTO, _('Auto-detect')),
```

Comment:
Maybe add a comment here to always keep `auto` on top since this drives the slicing logic in other places